ftests/nuxeo-platform-explorer-ftests/explorer-ftests-base/src/main/java/org/nuxeo/functionaltests/explorer/testing/AbstractExplorerTest.java

```diff
@@ -277,17 +277,12 @@ protected void switchBackToPreviousWindow() {
         }
     }
 
-    public static String getReferencePath(String path) throws IOException {
-        URL fileUrl = Thread.currentThread().getContextClassLoader().getResource(path);
-        if (fileUrl == null) {
-            throw new IllegalStateException("File not found: " + path);
-        }
-        return FileUtils.getFilePathFromUrl(fileUrl);
+    public static String getReferenceContent(Path path) throws IOException {
+        return IOUtils.toString(getReferenceStream(path), StandardCharsets.UTF_8).trim();
```

Comment:
You must close the stream, so this should be in a try-with-resources.
Check other callers of `getReferenceStream` as well, they're wrong too I think.